Scripts/RemediationScripts/Remediate-SetClusterProtectionLevelForServiceFabric.ps1

```diff
@@ -186,8 +199,11 @@ function Set-ClusterProtectionLeveltoEncryptandSignforServiceFabric
         {
             Write-Host "[Step 1 of 4] Validating and installing the modules required to run the script and validating the user..."
             Write-Host $([Constants]::SingleDashLine)
-            Write-Host "Setting up prerequisites..."
+            Write-Host "Setting up prerequisites..." -ForegroundColor $([Constants]::MessageType.Info)	
+            Write-Host $([Constants]::SingleDashLine)
             Setup-Prerequisites
+            Write-Host "Completed setting up prerequisites" -ForegroundColor $([Constants]::MessageType.Update)	
```

Comment:
We are going to follow the standard of ending sentences with '.'
Refer to App service PR for more clarity

Reply:
This is done, thanks